core/lib/zksync_core/src/state_keeper/updates/miniblock_updates.rs

```diff
@@ -19,7 +19,8 @@ pub struct MiniblockUpdates {
     pub executed_transactions: Vec<TransactionExecutionResult>,
     pub events: Vec<VmEvent>,
     pub storage_logs: Vec<StorageLogQuery>,
-    pub l2_to_l1_logs: Vec<L2ToL1Log>,
+    pub user_l2_to_l1_logs: Vec<L2ToL1Log>,
+    pub system_l2_to_l1_logs: Vec<L2ToL1Log>,
```

Comment:
Thinking out of loud, should we have different types for these logs. It's too easy to mess up with them

Reply:
+1 !